### ISSUE TITLE
Importing datajoint with latest numpy raises ABI version error

### ISSUE THREAD

**Issue by mspacek** [datajoint/datajoint-python#479]
After updating numpy to the latest 1.15.0 via pip, I get this error when importing datajoint (both the 0.10 release, as well as from git) in Python 3.5.2 and IPython 6.5.0 on xubuntu 16.04:
```python
/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
```
This is a known problem for some libraries compiled against an older version of numpy, probably libraries such as sklearn and pandas. I don't think datajoint is compiled against numpy, so I guess it imports from one or more of such libraries?

The warnings are apparently pretty benign, and easy to silence. From https://stackoverflow.com/a/40846742:

```python
import warnings
warnings.filterwarnings("ignore", message="numpy.dtype size changed")
warnings.filterwarnings("ignore", message="numpy.ufunc size changed")
```

I've added the above to our own datajoint project, but it would be nice if datajoint did this as well.

**Comment by dimitri-yatsenko** [comment]
This is expected to resolve itself in upcoming releases of scipy/numpy, correct?

**Comment by mspacek** [comment]
I have no idea. Do you have a reference for that?

**Comment by mspacek** [comment]
This appears to have been fixed in numpy 1.15.1
